cartoframes/context.py

```diff
@@ -249,11 +247,63 @@ def read(self, table_name, limit=None, decode_geom=False, shared_user=None):
         result = self.copy_client.copyto_stream(query)
         df = pd.read_csv(result)
 
+        return self._clean_DataFrame_from_carto(df, table_name, schema)
+
+    def _clean_DataFrame_from_carto(self, df, table_name, schema='public', decode_geom=False):
```

Comment:
mmm why not `_clean_data_frame_from_carto`?

Reply:
Because the class name is `DataFrame`, but ...